<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunConfiguration</class>
 <widget class="QWidget" name="RunConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="arrowImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="icons.qrc">:/icons/bullet_arrow_right.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>Run</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/small_gear.png</normaloff>:/icons/small_gear.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="detailsButton">
          <property name="text">
           <string>Details</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/small_wrench.png</normaloff>:/icons/small_wrench.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/small_delete.png</normaloff>:/icons/small_delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="detailsWidget" native="true">
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="configurationNameLabel">
           <property name="text">
            <string>Configuration Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="configurationNameEdit">
           <property name="text">
            <string>RunConfig</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="programLabel">
           <property name="text">
            <string>Program:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="programCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="targetGraphLabel">
           <property name="text">
            <string>Input Graph:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="targetGraphCombo"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>detailsButton</sender>
   <signal>pressed()</signal>
   <receiver>RunConfiguration</receiver>
   <slot>toggleDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>pressed()</signal>
   <receiver>RunConfiguration</receiver>
   <slot>runConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleDetails()</slot>
  <slot>runConfiguration()</slot>
  <slot>viewResult()</slot>
 </slots>
</ui>
